frame/staking/src/mock.rs

```diff
@@ -269,6 +269,8 @@ where
 }
 
 pub type Extrinsic = TestXt<Call, ()>;
+pub(crate) type StakingCall = crate::Call<Test>;
+pub(crate) type TestRuntimeCall = <Test as frame_system::Config>::Call;
```

Comment:
we should really just rename the `Call` generated by the construct_runtime! to `OuterCall`.

Reply:
(not realted to this PR, I am just ranting)

Reply:
Is there an issue open for this / should we open an issue?

Reply:
no issue afaik.